crypto/keys/secp256k1/secp256k1_cgo.go

```diff
@@ -1,4 +1,5 @@
-// +build libsecp256k1
+//go:build libsecp256k1_sdk
```

Comment:
Im leaning towards pushing towards a single cgo flag that we group other cgo deps on.